docs/tutorials/10-minute-walkthrough/index.md

```diff
@@ -107,18 +82,13 @@ Same as the previous step, but we'll output to ElasticSearch now.
 
 ### Run it
 
-    bin/logstash agent -f apache-elasticsearch.conf web
+    bin/logstash agent -f apache-elasticsearch.conf
```

Comment:
Well, I _was_ only supposed to prune web and kibana, but I can remove agent, I suppose.

Also, isn't the 10-minute-walkthrough supposed to go the way of the dodo, too?  The non-asciidoc docs are all deprecated, aren't they?